.github/filters.yaml

```diff
@@ -1,26 +1,37 @@
+# This file determines when jobs should run or skipped based on which files were modified
+
 backend:
-  - '.github/actions/yarn-nm-install/*.yml'
-  - '.github/workflows/**'
-  - 'packages/**/package.json'
   - 'packages/**/server/**/*.(js|ts)'
   - 'packages/**/strapi-server.js'
   - 'packages/{utils,generators,cli,providers}/**'
   - 'packages/core/*/{lib,bin,ee,src}/**'
   - 'packages/core/database/**'
-  - 'yarn.lock'
+  - 'jest-preset.back.js'
 frontend:
-  - '.github/actions/yarn-nm-install/*.yml'
-  - '.github/workflows/**'
-  - 'packages/**/package.json'
   - 'packages/**/admin/src/**'
   - 'packages/**/admin/ee/admin/**'
   - 'packages/**/strapi-admin.js'
   - 'packages/admin-test-utils/**'
-  - 'yarn.lock'
+  - 'jest-preset.front.js'
+  - 'fileTransformer.js'
 api:
   - 'tests/api/**'
 e2e:
   - 'tests/e2e/**'
   - 'playwright.base.config.js'
 cli:
   - 'tests/cli/**'
+
+# TODO: this needs a lot more in it
```

Comment:
but it's an improvement over what we had, so we can build it up over time if we notice tests are skipped when they shouldn't be